ModelBundle/Resources/translations/validators.fr.yml

```diff
@@ -4,3 +4,4 @@ open_orchestra_model_validators:
     document:
         content_type:
             unique_field_id: Les identifiants des champs doivent être unique
+            unique_content_type_id: L'identifiant du type de contenu doit être unique
```

Comment:
As it's an error message, maybe you could be more directive, with a message like "This id is already in use, please choose another one"